decidim-admin/app/forms/decidim/admin/participatory_process_form.rb

```diff
@@ -25,7 +25,7 @@ class ParticipatoryProcessForm < Form
       attribute :slug, String
       attribute :hashtag, String
       attribute :promoted, Boolean
-      attribute :scope_id, Integer
+      attribute :scope_id, String
```

Comment:
I'm seeing that sometimes you will receive the string "global" here, and thus you rely on `global.to_i` being `0` and the ActiveRecord scope being empty (which is what we want, I guess). Is that it?